cmd/kube-proxy/app/conntrack.go

```diff
@@ -97,7 +97,7 @@ func (realConntracker) setIntSysCtl(name string, value int) error {
 
 	sys := sysctl.New()
 	if val, _ := sys.GetSysctl(entry); val != value {
-		klog.Infof("Set sysctl '%v' to %v", entry, value)
+		klog.InfoS("Set sysctl", entry, value)
```

Comment:
```suggestion
		klog.InfoS("Set sysctl", "entry", entry, "value", value)
```

Reply:
why is this marked as resolved if the suggestion wasn't accepted?
which one is the preferred?

Reply:
the kep seems to prefer the keysAndValues format

Reply:
ouch! Could you take a look again,@yxxhero?